content/Konveyor/installation.md

```diff
@@ -22,43 +22,32 @@ Follow the steps below to provision minikube for single users deploying Konveyor
 ```
 3. Install Operator Lifecycle Manager (OLM), a tool to help manage the Operators running on your cluster.
 ```
-[user@user ~]$ curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.21.2/install.sh | bash -s v0.21.2
+[user@user ~]$ curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.24.0/install.sh | bash -s v0.24.0
 ```
 
 ## Installing Konveyor Operator
 Operators are a structural layer that manages resources deployed on Kubernetes (database, front end, back end) to automatically create a Konveyor instance instead of doing it manually.
 
-**Requirements**
-Konveyor requires a total of 5 persistent volumes (PVs) used by different components to successfully deploy, 3 RWO volumes and 2 RWX volumes will be requested via PVCs.
-
-|Name|Default Size|Access Mode|Description|
-|--|--|--|--|
-|hub database|5Gi|RWO|Hub DB|
-|hub bucket|100Gi|RWX|Hub file storage|
-|keycloak postgresql|1Gi|RWO|Keycloak backend DB|
-|pathfinder postgresql|1Gi|RWO|Pathfinder backend DB|
-|maven|100Gi|RWX|maven m2 repository|
-
-Follow the steps below to install the Konveyor Operator in the konveyor-tackle namespace (default) on any Kubernetes distribution, including minikube.
+Follow the steps below to install the Konveyor Operator in the `my-konveyor-operator` namespace (default) on any Kubernetes distribution, including minikube.
 
 **Procedure**
-1. Install the Konveyor Operator.
+1. Install the latest released Konveyor Operator.
 ```
-[user@user ~]$ kubectl apply -f https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml
+[user@user ~]$ kubectl create -f https://operatorhub.io/install/konveyor-0.1/konveyor-operator.yaml
 ```
-This step will create the konveyor-tackle namespace, catalogsource and other OLM related objects.
-
-### Installing _beta_ (or special branches)
+This step will create the `my-konveyor-operator` namespace, catalogsource and other OLM related objects.
 
-If you need to deploy a beta release (or special branch) please replace the *main* branch in URL with the desired beta branch (i.e v2.0.0-beta.0):
+### Installing the latest
 
-`$ kubectl apply -f https://raw.githubusercontent.com/konveyor/tackle2-operator/<release>-beta.0/tackle-k8s.yaml`
+If you need to deploy a latest build off main please use the below url
 
-**Note:** Upgrades between beta releases are **not guaranteed** , once installed, we strongly suggest to edit your subscription and switch to Manual upgrade mode for beta releases: `$ kubectl edit subscription` -> installPlanApproval: Manual
+```
+[user@user ~]$ kubectl create -f https://operatorhub.io/install/alpha/konveyor-operator.yaml
+```
```

Comment:
alpha isn't really 'latest' tracking main, alpha are more pre-release builds.

To install ':latest' which tracks main we can use:
https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml


The diff is that we need to create a custom CatalogSource using
quay.io/konveyor/tackle2-operator-index:latest

```
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: konveyor
  namespace: konveyor-tackle
spec:
  displayName: Konveyor Operator
  publisher: Konveyor
  sourceType: grpc
  image: quay.io/konveyor/tackle2-operator-index:latest
```

and then a Subscription using the custom catalog source

```
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: konveyor-operator
  namespace: konveyor-tackle
spec:
  channel: development
  installPlanApproval: Automatic
  name: konveyor-operator
  source: konveyor
  sourceNamespace: konveyor-tackle


```